dependencies/pom.xml

```diff
@@ -955,6 +972,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
```

Comment:
Same here, this dependency should not be here

Reply:
Done